### PR TITLE
Added volumesFrom to Docker template options

### PR DESCRIPTION
**Pull request by grkvlt**
See #252 for more information.

### CODE REVIEW

**Comment by grkvlt** [comment]
@andreaturli @nacx can you please review this and #252 which update the available options for Docker

File: docker/src/main/java/org/jclouds/docker/domain/Config.java [hunk 1]
```diff
@@ -117,14 +117,14 @@ public static Config create(String hostname, String domainname, String user, int
...
-              copyWithNullOf(entrypoint), image, copyOf(volumes), workingDir, networkDisabled,
+              copyWithNullOf(entrypoint), image, copyWithNullOf(volumes), workingDir, networkDisabled,
               copyOf(exposedPorts), copyOf(securityOpts), hostConfig,
-              copyOf(binds), copyOf(links), copyOf(lxcConf), copyOf(portBindings), publishAllPorts, privileged,
-              copyWithNullOf(dns), dnsSearch, volumesFrom, copyOf(capAdd), copyOf(capDrop), copyOf(restartPolicy));
+              copyWithNullOf(binds), copyWithNullOf(links), copyOf(lxcConf), copyOf(portBindings), publishAllPorts, privileged,
+              copyWithNullOf(dns), copyWithNullOf(dnsSearch), copyWithNullOf(volumesFrom), copyWithNullOf(capAdd), copyWithNullOf(capDrop), copyOf(restartPolicy));
    }
 
    public static Builder builder() {
```
Comment (andreaturli) [inline docker/src/main/java/org/jclouds/docker/domain/Config.java]: [minor] it would be nice to use Auto Value builder available since 1.1 https://github.com/google/auto/tree/master/value#builders

File: docker/src/main/java/org/jclouds/docker/domain/HostConfig.java [hunk 2]
```diff
@@ -79,16 +78,16 @@ public Builder toBuilder() {
    public static final class Builder {
```
Comment (andreaturli) [inline docker/src/main/java/org/jclouds/docker/domain/HostConfig.java]: same here, see https://github.com/google/auto/tree/master/value#builders

**Comment by andreaturli** [comment]
thanks @grkvlt looks good, just a couple of minor comments

File: docker/src/main/java/org/jclouds/docker/domain/Config.java (hunk 1 above)
Comment (grkvlt) [inline docker/src/main/java/org/jclouds/docker/domain/Config.java]: Yes, but moving to AutoValue builders should be a separate PR, I think

**Comment by grkvlt** [comment]
@andreaturli if you're OK with these PRs I suggest I could make a _separate_ patch using the AutoValue builders, since it won't be adding any functionality. That way these can be merged sooner. WDYT?

**Comment by andreaturli** [comment]
Sounds good to me. I'll take care of the builders later

**Comment by andreaturli** [comment]
thanks @grkvlt, merged at [1.9.x](http://git-wip-us.apache.org/repos/asf/jclouds-labs/commit/dbea9c3e)
